pkg/app/consume.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/grafana/prometheus-pulsar-remote-write/pkg/metrics"
```

Comment:
nit: Seems like the golangci-lint is not enabled for checking on that, but usually we try to follow the import ordering as described here:  https://github.com/uber-go/guide/blob/master/style.md#import-group-ordering

Reply:
Will change